### PR TITLE
Refactor Integration tests

### PR DESCRIPTION
**Pull request by dgarros**
- Install Netbox Collection from Github instead of Galaxy
- Enhance integration tests to support multiple version of Netbox
- Run integration tests for Netbox 2.8 and 2.9

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,10 +50,29 @@ jobs:
...
         - "poetry install"
+      env: "NETBOX_VERSION=v2.10"
+
+    - stage: "integration"
+      name: "Netbox v2.9"
+      before_script:
+        - "pip install invoke toml poetry ansible pynetbox"
+        - "ansible-galaxy collection install git+https://github.com/netbox-community/ansible_modules.git,v1.2.0"
+        - "poetry install"
+      env: "NETBOX_VERSION=v2.9"
```
Comment (jvanderaa) [inline .travis.yml:66]: Do we need a `script` here as well?

**Review by jvanderaa — COMMENTED** [review]
Quick question on the script portion for NetBox 2.9 and 2.10

File: .travis.yml (hunk 1 above)
Comment (dgarros) [reply .travis.yml]: Good catch, I've made the update and after multiple tries the integration tests are now passing for 2.8, 2.9 and 2.10

**Review by jvanderaa — APPROVED** [review]
LGTM

**Pull request merged**